packages/connect/src/connector/smoldot-light.ts

```diff
@@ -238,6 +243,8 @@ export const createScClient = (config?: Config): ScClient => {
   const addWellKnownChain: AddWellKnownChain = async (
     supposedChain: WellKnownChain,
     jsonRpcCallback?: (msg: string) => void,
+    potentialRelayChains?: Chain[],
```

Comment:
since "well known chains" are all relay-chains, why would we need this? 🤔

Reply:
will remove because this is not needed for wellKnownChains